packages/webapp/.storybook/main.cjs

```diff
@@ -1,24 +1,29 @@
 module.exports = {
```

Comment:
I can't find the difference, what's changed? 😅

Reply:
Its hard to tell but I removed the` /stories` after src in `stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],`. It was matching all .stories files in the stories directory now it will look for them in src